qutebrowser/utils/version.py

```diff
@@ -46,14 +46,19 @@
 
 try:
     from PyQt5.QtWebEngineWidgets import QWebEngineProfile
-except ImportError:  # pragma: no cover
+except ImportError:
```

Comment:
Why? This indeed isn't covered (with QtWebEngine), so it [breaks](https://travis-ci.org/qutebrowser/qutebrowser/jobs/508891367#L571) the coverage checks.